games-misc/cowsay/cowsay-3.04.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="configurable talking ASCII cow (and other characters)"
+HOMEPAGE="https://github.com/tnalpgge/rank-amateur-cowsay"
+SRC_URI="https://github.com/tnalpgge/rank-amateur-${PN}/archive/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~mips ~ppc64 ~x86 ~x64-solaris"
+
+RDEPEND=">=dev-lang/perl-5"
+DEPEND="${RDEPEND}"
```

Comment:
Under EAPI 7, Perl would be a `BDEPEND` rather than a `DEPEND`.

Reply:
fixed